src-tauri/src/main.rs

```diff
@@ -50,19 +60,34 @@ async fn initialize_app(app: AppHandle) -> Result<(), String> {
         }
     };
 
-    let file_info = get_platform_flash_runtime(&env::consts::OS)?;
-    if !file_info.0.exists() {
+    let appdata_path = get_appdata_path(&app);
+
+    // Creates AppData folder if it does not exist
+    if !get_appdata_path(&app).exists() {
+        fs::create_dir_all(&appdata_path).expect("Could not create appdata folder!");
+    }
```

Comment:
> Could we just update the ensure folder exists inside of download_runtime instead of doing this ad hoc?
Feel free to do so, but since this will only be run once ever™ (during the first time the player uses the Launcher on his PC) I don't think its worth it.

> sometimes the dev builds have loads of permissions but the prod ones wont let you access a directory.

I've checked with both Dev and Release builds and I didn't get any issues. I don't think there should be any permission problems as this is a "standard" implemented by Tauri as seen [here](https://v1.tauri.app/v1/api/js/path/#appdatadir) and [here](https://docs.rs/tauri/1.8.1/tauri/struct.PathResolver.html#method.app_data_dir)